libs/components/lookup/src/lib/modules/lookup/lookup.component.spec.ts

```diff
@@ -3776,7 +3782,9 @@ describe('Lookup component', function () {
 
         fixture.detectChanges();
         await fixture.whenStable();
-        await expectAsync(document.body).toBeAccessible(axeConfig);
+        await expectAsync(
+          fixture.nativeElement.ownerDocument.body
```

Comment:
Can you provide some background on this change?

Reply:
I made this change when I wasn't seeing the result I expected, it ended up being a different issue, and I've reverted this change.